lib/src/models/message.dart

```diff
@@ -14,10 +14,18 @@ class Message {
   /// Whether the message was sent by the Agent.
   bool get isRemote => !isLocal;
 
+  /// The File Id of the stored file within history.
```

Comment:
As someone consuming this API, the "stored file within history" bit is confusing.